Externals/libusb/CMakeLists.txt

```diff
@@ -82,41 +77,48 @@ if(UNIX)
     if(CMAKE_THREAD_LIBS_INIT)
       target_link_libraries(usb PRIVATE "${CMAKE_THREAD_LIBS_INIT}")
     endif()
-    set(THREADS_POSIX TRUE)
-elseif(WIN32)
-    target_sources(usb PRIVATE
-        libusb/os/poll_windows.c
-        libusb/os/threads_windows.c
-    )
+    set(PLATFORM_POSIX TRUE)
 endif()
 
+# Create config.h
+
 include(CheckFunctionExists)
 include(CheckIncludeFiles)
+#include(CheckLibraryExists)
```

Comment:
Is there a reason this was left in but commented out?

Reply:
please make further comments about libusb specific stuff on https://github.com/dolphin-emu/dolphin/pull/10577 

in this case, the udev stuff was left "in order" (relative to the config.h.in file) but commented out because it's actually filled out manually, higher in the cmake file. idk, i thought it made sense :D

Reply:
Oh I'm so sorry, I did not realize there was another PR that this was built on.